cypress/e2e/filter/filterCases.cy.ts

```diff
@@ -105,44 +106,79 @@ describe("Case list", () => {
 
       cy.contains("Exceptions")
 
-      collapseFilterSection("Reason", "Exceptions")
-      expandFilterSection("Reason", "Exceptions")
+      collapseFilterSection("Reason", "#exceptions-type")
+      expandFilterSection("Reason", "#exceptions-type")
     })
 
-    it("Should expand and collapse court date filter navigation", () => {
+    it("Should expand and collapse court date filter navigation with the ratio conditional sections collapsed after the second expand", () => {
       visitBasePathAndShowFilters()
 
       cy.contains("Date range")
 
-      collapseFilterSection("Court date", "Date range")
-      expandFilterSection("Court date", "Date range")
+      cy.contains("Court date").parent().parent().parent().find("button").click()
+      cy.get("#date-range").should("not.be.visible")
+      expandFilterSection("Court date", "#date-range")
+
+      // Custom date range & date range are collapsed
+      cy.get("#date-from").should("not.be.visible")
+      cy.get("#date-range-yesterday").should("not.be.visible")
+      // Opening custom date range collapses date range & opens custom date range
+      cy.get("#custom-date-range").click()
+      cy.get("#date-from").should("be.visible")
+      cy.get("#date-range-yesterday").should("not.be.visible")
+    })
+
+    it("Should remove the selection of the date range when it's been changed to the custom date range", () => {
```

Comment:
I think we also need to test for the opposite:
"Should remove the selection of the custom date range when it's been changed to named date range"

![image](https://user-images.githubusercontent.com/22743709/221171239-ca0c8f45-bb21-43d9-87eb-610386cc2b3f.png)

Reply:
It's a great suggestion! I've added the test. Thanks!